### ISSUE TITLE
Optional MultiValue usage in KQL

### ISSUE THREAD

**Issue by rbanks54** [microsoft/Application-Insights-Workbooks#1617]
I have a workbook with a global parameter for holding selections made from an initial query table. I then want to use that selection to filter results in a later query.

I also want to default to showing all data when no selection has been made, and only filter when the multi-value parameter is populated.

I've set set the initial query to export it's selection parameters as follows:

![image](https://user-images.githubusercontent.com/234581/133022174-b29bc22e-dfdf-463e-8a04-e2246cbaa18a.png)

I see results when I have made a selection, but I can't figure out how to create value KQL when the parameter has no selections, and is a null.

For example:

`| where AppService has_any ({AppServices})`
is rendered as
`| where AppService has_any ()`

and similarly something like this to try and work around the null doesn't parse either:
`| extend myFilter = iff({AppServices:label} = '<unset>','',{AppServices})`
because it's rendered as
`| extend myFilter = iff('<unset>' = '<unset>','',)`

Basically, I need a value other than `:label` that won't be nulll when the parameter is unset. Are there any tips for doing this? I can't see anything in the docs that helps me and I can't figure out how to create KQL that will be valid when the the parameter is null.
